sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidOperatorTable.java

```diff
@@ -243,6 +253,8 @@
           .add(new MultiValueStringPrependOperatorConversion())
           .add(new MultiValueStringSliceOperatorConversion())
           .add(new StringToMultiValueStringOperatorConversion())
+          //
```

Comment:
Stray `//` -- perhaps should be `// IPv4 operators`

Reply:
Yeah, I can add that heading though it's a bit redundant with the named constant. I can make things consistent by either adding each of the IPv4 operators here or by creating new named constants for each of the groups.

Reply:
Oh, I see, I guess you were doing this on purpose to separate the sections without adding a header. I had assumed it was a mistake.

The list is getting really long. I suppose it's a good time to break the groups out into their own named constants.